pallets/process-validation/src/lib.rs

```diff
@@ -29,11 +30,29 @@ impl Default for ProcessStatus {
     }
 }
 
-#[derive(Encode, Decode, Default, Clone, PartialEq)]
+#[derive(Encode, Decode, Clone, PartialEq)]
+#[cfg_attr(feature = "std", derive(Debug))]
+pub struct Version {
+    version: i32, // TODO: sort this type, should be included from trait
+}
+// TODO remove once type has been soprted <version>
+impl EncodeLike<i32> for Version {}
+
+
+#[derive(Encode, Decode, Clone, PartialEq)]
```

Comment:
You know this isn't need, but just a reminder that once the version is swapped out this can be removed

Reply:
Yes, so this is like a very draft that is not really there, just wanted share something with out and get some input

Replied to Dan's comment above in regards to the version type